app/components/IdentityCard/FailedIdentityDetails.tsx

```diff
@@ -17,25 +19,66 @@ export default function FailedIdentityDetails({
 }: FailedIdentityDetailsProps) {
     const sessionId = getSessionId(identity);
     const mail = 'concordium-idiss@notabene.id';
-    const subject = `Reference: ${sessionId}`;
+    const cc = 'idiss@concordium.software';
+    const subject = `Issuance Reference: ${sessionId}`;
+    const body = `Hi! My identity issuance failed.
+
+Here is my info:
+
+Issuance reference:
+${sessionId}
+
+Desktop Wallet version:
+v${pkg.version}
+
+Operating system:
+${window.platform}`;
```

Comment:
On mac, it shows "darwin" as operating system in the failed info. Maybe it would be nice to translate this to "MacOS"? windows will also show as "win32", which would also be good to translate, if we decide it's something we want to do?

Reply:
I've translated them.